rpa-recorder/6.1/modules/ROOT/pages/user-interface.adoc

```diff
@@ -114,6 +114,46 @@ To continue recording, add a Bot Task.
 [NOTE]
 Pause the recording before adding an activity or changing the creation mode of the activities. If you open the pop-up using the respective hotkey, while the recording is running, the recording automatically pauses.
 
+== Web Recording Browser
+
+Use the Web Recording Browser to record website navigation actions. It behaves as a regular web browser and enables you to:
+
+* Automatically identify elements of a website with which you interact and determine their XPath and attributes.
+* Save your attribute selection of a web element to reuse it when recording the same web elements on a website.
+* Record mouse hover positioning during web navigation.
+
+Adding a Web Task opens the Web Recording Browser. When you add more than one Web Task, you can select *Continue with already existing web session* to display the last loaded page from the previous Web Task and continue from that point.
+Otherwise, you can enter a URL in the address bar of the browser, and it loads this URL when you press Enter or click the image:play-icon.png[Play button].
+
+image:web-recording-browser-blank.png[Web Recording Browser]
+
+=== Select Web Elements
+
+When you hover the cursor over a web element, the browser highlights it with a blue background. After highlighting an element, its XPath and available attributes show on the Attribute Selection panel.
+
+To record the mouse hover event, press `F2`.
```

Comment:
Suggestion to add to this sentence:
"To record the mouse hover event **on a highlighted element**, press `F2`."